src/hotspot/share/runtime/objectMonitor.inline.hpp

```diff
@@ -31,7 +31,7 @@
 #include "runtime/synchronizer.hpp"
 
 inline intptr_t ObjectMonitor::is_entered(TRAPS) const {
-  if (THREAD == _owner || THREAD->is_lock_owned((address) _owner)) {
+  if (THREAD == owner_raw() || THREAD->is_lock_owned((address) owner_raw())) {
```

Comment:
Suggestion: add a local to avoid two calls to `owner_raw()`.

Reply:
Done.